MagazineLayout/LayoutCore/ModelState.swift

```diff
@@ -209,9 +209,7 @@ final class ModelState {
     }
 
     let maxY = cachedMaxYForSection(atIndex: targetSectionIndex) ?? sectionMaxY
-    if !disableSectionMaxYsCache {
-      cacheMaxY(maxY, forSectionAtIndex: targetSectionIndex)
-    }
```

Comment:
I meant to delete this in my last PR - this extra property / check isn't necessary, and was never actually used